src/components/molecules/CarouselText/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './style.css';
+import Title from '../../atoms/Title/index';
+
+function CarouselText() {
+    return (
+        <div className="carousel__text">
+            <Title className="title__carousel" title='Made with love' />
+            <Title className="subtitle__carousel" title='for you' />
```

Comment:
create subbtitle component or pass subtitle as prop like `isSubtitle="true" ` or `titleVariant="subtitle"` ifyou want it as reusable component

Reply:
ok, changed it to titleVariant